src/signatures/secp256k1_ecdsa.rs

```diff
@@ -0,0 +1,198 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use core::convert::TryFrom;
+
+use zeroize::ZeroizeOnDrop;
+
+pub const SECRET_KEY_LENGTH: usize = 32;
+pub const PUBLIC_KEY_LENGTH: usize = 33;
+pub const SIGNATURE_LENGTH: usize = 65;
+pub const ADDRESS_LENGTH: usize = 20;
```

Comment:
It's usually a good thing to have these as associated const of the types, so we don't have to import them when we need them.

Reply:
Good point. Those are not associated const for Ed25519. Shall we make them so there too?

Reply:
I'd do it, yeah

Reply:
I'd say we should keep global consts deprecated too. Otherwise we might break too much

Reply:
What would be a good name for associated consts: SecretKey::LENGTH?

Reply:
Yeah I think that's enough 👍🏻

Reply:
Done. Use XXX::LENGTH consts for secp256k1_ecdsa/ed25519 types, deprecated ed25519::XXX_LENGTH consts.

Reply:
Done